tests/test-data/pkg/expected-warnings-darwin.txt

```diff
@@ -68,6 +64,10 @@
   The addon must be distributed with executable as %2.
   %1: node_modules/fsevents/fsevents.node
   %2: path-to-executable/fsevents.node
+> Warning Cannot include addon %1 into executable.
+  The addon must be distributed with executable as %2.
+  %1: node_modules/xxhash/build/Release/hash.node
+  %2: path-to-executable/hash.node
```

Comment:
Same contents, in different order. To do: Improve the `pkg` diff comparison to accept same contents in different order.

Reply:
Maybe just a list of the paths, ie the `node_modules/xxhash/build/Release/hash.node` bits?